packages/requests/src/lib/requests-result.ts

```diff
@@ -196,6 +204,7 @@ export function trackRequestResult<TData>(
                 isLoading: false,
                 isSuccess: false,
                 status: 'error',
+                lastRequestTime: Date.now(),
```

Comment:
The name is a bit confusing because it's not the time of the request. It's time the request ends.

Reply:
Yeah you're right.
- lastRequestResponseTime
- lastRequestEndTime

Do you have another idea? I think lastRequestResponseTime is good.

Reply:
FIxed it